.travis.yml

```diff
@@ -14,10 +14,6 @@ notifications:
   email: false
 after_success:
   - "bin/coveralls"
-cache:
-  directories:
-    - eggs
-    - node_modules
```

Comment:
I did remove all caches. We suspected and issue related to caching an this pull-request is for testing travis.